Makefile

```diff
@@ -127,10 +126,10 @@ bench-tx: ## Run transaction benchmarks
 
 # --- installing ----------------------------------------------------------------------------------
 
-.PHONY: install-tx-prover
-install-tx-prover: ## Install transaction prover's CLI
-	cargo install --path bin/tx-prover --bin miden-tx-prover --locked --features concurrent
+.PHONY: install-proving-service
+install-proving-service: ## Install proving service's CLI
+	cargo install --path bin/proving-service --bin miden-proving-service --locked --features concurrent
 
-.PHONY: install-tx-prover-testing
-install-tx-prover-testing: ## Install transaction prover's CLI intended for testing purposes
-	cargo install --path bin/tx-prover --bin miden-tx-prover --locked --features concurrent,testing
+.PHONY: install-proving-service-testing
+install-proving-service-testing: ## Install proving service's CLI intended for testing purposes
+	cargo install --path bin/proving-service --bin miden-proving-service --locked --features concurrent,testing
```

Comment:
As I mentioned in one of the other comments, I don't think this will be needed any longer.